src/circuit/Circuit1.java

```diff
@@ -1 +1,167 @@
 package circuit;
+
+import java.util.ArrayList;
+
+/**
+ * {@code Circuit} represented as a {@code ArrayList} with implementations of
+ * primary methods.
+ *
+ * @convention <pre>
+ * [first value within $this.rep > 0] and
+ * [this.object int >= 50 means object is resistor] and
+ * [starting voltage is 5 unless specified in full args constructor]
+ * </pre>
+ *
+ * @correspondence <pre>
+ * [values are input as 1s when generating circuit]
+ * </pre>
+ *
+ * @author Caden Hansen
+ *
+ */
+public class Circuit1 extends CircuitSecondary {
+
+    /**
+     * Representation of {@code this}.
+     */
+    private ArrayList<Integer> rep;
+
+    /**
+     * Representation of initial voltage.
+     */
+    private int voltage;
+
+    /**
+     *
+     */
+    private void createNewRep() {
+
+        this.rep = new ArrayList<Integer>();
+        this.voltage = 5;
+        this.rep.add(5);
+
+    }
+
+    /**
+     * Default constructor.
+     */
+    public Circuit1() {
+
+        this.createNewRep();
+
+    }
+
+    /**
+     *
+     * @param voltage
+     *            voltage
+     *
+     *            Full argument constructor.
+     *
+     */
+    public Circuit1(int voltage) {
+
```

Comment:
Do you have any preconditions for the voltage? If so, I would add an assert here.